terraform/gcp/projects/daskhub-template.tfvars

```diff
@@ -8,7 +8,7 @@
 prefix     = "{{ cluster_name }}"
 project_id = "{{ project_id }}"
 
-zone   = "{{ cluster_region }}"
+zone   = "{{ cluster_region }}-b"
```

Comment:
Quick question in a high-level post review (so I might be totally wrong :wink:): do we always want to deploy to the `-b` zone? Should not get the `zone` from `foo_cluster.tfvars` file instead of "building it" from the region?
cc @GeorgianaElena

Reply:
> Should not get the zone from foo_cluster.tfvars file instead of "building it" from the region?

@damianavila, so this template is actually used by the deployer to generate a `foo_cluster.tfvars` file. This initial file generated by the deployer is more of a starting point with some "default" basics that are meant to be edited to fulfill all of the needs for the cluster.

> do we always want to deploy to the -b zone?

Not necessarily, but it is my understanding that this is the go-to zone for us-west2 (which I believe is the default region if none is requested) if we want to be able to have GPU machines (which for a daskhub is probable) available and a vaster CPU types machines, including AMDs.

Reply:
>@damianavila, so this template is 

I knew I was missing something (well-placed "so I might be totally wrong 😉" message).
Thanks for the clarification!!

> Not necessarily,

Yep, it makes sense now that I actually realize what I am seeing 😉
Again, thanks for your patience and feedback.